pkg/cluster/cluster.go

```diff
@@ -760,6 +763,17 @@ func (c *Cluster) initHumanUsers() error {
 	return nil
 }
 
+func (c *Cluster) initHumanUsers() error {
+
+	for _, postgresAdminTeam := range c.OpConfig.PostgresAdminTeams {
+		c.initTeamMembers(postgresAdminTeam, true /*isPostgresAdminTeam*/)
+	}
+
+	c.initTeamMembers(c.Spec.TeamID, false /*isPostgresAdminTeam*/)
```

Comment:
What if your team is already listed in both the admin and the owner team, would resolveTeamMembers handle this correctly?